meilisearch/client.py

```diff
@@ -1,5 +1,9 @@
 from typing import Any, Dict, List, Optional
 
+import base64
+import hashlib
+import hmac
```

Comment:
Do the python users get these libs for free? Or do they need to install a native package?

Reply:
I don't have to add it to `Pipfile.lock` so I suppose they are native to python

Reply:
@alallema is correct, they are part of the standard library so free.